### DISCUSSION TITLE
Independent volume adjustment per source when grouped

### DISCUSSION THREAD

**Discussion by timmelander** [music-assistant/support#714] [Q&A] [answered]
I love the group feature because I use Sonos and I can stream a source to multiple Sonos speakers. However, I wish this had the same feature as the Sonos app to control the volume at each speaker. I can't find an option to separate a volume for each speaker when grouped.

**Comment by marcelveldt** [comment, marked as answer]
What do you mean?
This is already how it is implemented. 

<img width="944" alt="Schermafbeelding 2022-07-23 om 20 25 41" src="https://user-images.githubusercontent.com/6389780/180618157-427c4d1f-7e2b-4870-b495-78f757b439f5.png">
